examples/gadgets/withfilter/trace/network/network.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/cilium/ebpf/rlimit"
+	containercollection "github.com/inspektor-gadget/inspektor-gadget/pkg/container-collection"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/container-collection/networktracer"
+	containerutils "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/types"
+	tracernetwork "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/trace/network/tracer"
+	tracernetworktypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/trace/network/types"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators/kubeipresolver"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators/kubenameresolver"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/socketenricher"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/host"
+)
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %s", err.Error())
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	if err := rlimit.RemoveMemlock(); err != nil {
+		return fmt.Errorf("failed to remove memory limit: %v", err)
```

Comment:
nit: remove the failed to, and similar prefixes on the error messages.

See this semgrep rule:https://github.com/inspektor-gadget/inspektor-gadget/blob/e3f75523c0d73e8685b4b908a710d2b27a472445/.semgrep_rules.yaml#L2-L7

Reply:
Done